.github/workflows/doc.yml

```
@@ -32,6 +33,37 @@ jobs:
     - uses: actions/checkout@v3
     - name: Register Sphinx problem matcher
       run: echo "::add-matcher::.github/problem-matchers/sphinx.json"
+    - name: 'Setup Python'
+      uses: actions/setup-python@v3
```

We can bump to the new v4
```suggestion
      uses: actions/setup-python@v4
```

---

I entirely missed v4 was even released!

---

Oh, and it needs a `python-version`, can be `"3.x"`:

https://github.com/actions/setup-python/releases/tag/v4.0.0

---

Jinx https://github.com/python/cpython/pull/93736#issuecomment-1153009014 :P